src/source.css

```diff
@@ -13464,6 +13483,72 @@ path[fill="rgb(255,216,54)"] {
     opacity: 0.6 !important;
 }
 
+/* Activities bg */
+.container_c7c1c7 {
+	background-color: var(--dark-bg) !important;
+}
+
+/* Activities app names */
+.appDetailsContainer_e5da8b {
+	background-color: var(--hover-color) !important;
+}
+
+/* Eye button in activities */
+.appDetailsHeaderContainer_e5da8b button svg path {
+    fill: var(--soft-bright-color) !important;
+}
+
+/* Activity description */
+.lineClamp1_a595eb span {
+    color: var(--soft-bright-color) !important;
+}
+
+/* bg on thing when pressed the activity */
+.container_ec8cd2.none_c49869.scrollerBase_c49869.fade_c49869 {
+	background-color: var(--dark-bg) !important;
+}
+
+/* Video bg in activity */
+.overviewContainerWithVideo_e80fe9 {
+	background-color: var(--hover-color) !important;
+}
+
+/* Tags in activity */
+.tagContainer_e80fe9 {
+	background-color: var(--hover-color) !important;
+}
+
+/* Buttons in activities */
+.container_e33430 .clickable_e33430 svg path,
+.clickable_e82700 svg path {
+    fill: var(--soft-bright-color) !important;
+}
+
+/* Authorize app */
+.content_c5a065.contentBackground_c5a065,
+.footer_c5a065 {
+	background-color: var(--dark-bg) !important;
+}
+
+/* Click to see command in activity */
+.messageContentWrapper_dbb3d0 {
+	background-color: var(--hover-color) !important;
+}
+	
+/* Authorize app circles */
+.scopes_cf14a8 svg path {
+    fill: var(--soft-bright-color) !important; /* Change color as needed */
+}
+/* Authorize app circles */
+.scopes_cf14a8 svg path {
+    fill: var(--soft-bright-color) !important; /* Change color as needed */
+}
+
+/* Apps hover popup */
+.container_b572de {
+	background-color: var(--dark-bg) !important;
+}
+
```

Comment:
Same as for the change above, just consolidate everything into one section, without the comments for every element. The main thing though, an "Activities" section [already exists here](https://github.com/Aely0/Kyanite/blob/fee4b16cd125bd5ab1d50afffd13a39069167fe8/src/source.css#L10435), meld this section to the already existing one.